src/scenes/CreateEvent/components/CreateEventCreatorPicker/index.js

```diff
@@ -30,18 +41,38 @@ class CreateEventCreatorPicker extends React.PureComponent {
       this.props.changeFormFieldValue(name, this.props.lastUsedAddress);
     }
 
-    return (<Select
-      {...input}
-      {...custom}
-      fullWidth
-    >
-      {this.props.walletAddresses.map((item, index) => (
-        <MenuItem key={item.address} value={item.address}>
-          {`${item.address}`}
-          {` (${item.qtum ? item.qtum.toFixed(2) : 0} QTUM, ${item.bot ? item.bot.toFixed(2) : 0} BOT)`}
-        </MenuItem>
-      ))}
-    </Select>);
+    return (<FormControl fullWidth>
+      <Select
+        {...input}
+        {...custom}
+        fullWidth
+        error={Boolean(error)}
+      >
+        {this.props.walletAddresses.map((item, index) => (
+          <MenuItem key={item.address} value={item.address}>
+            {`${item.address}`}
+            {` (${item.qtum ? item.qtum.toFixed(2) : 0} QTUM, ${item.bot ? item.bot.toFixed(2) : 0} BOT)`}
+          </MenuItem>
+        ))}
+      </Select>
+      {
+        error ?
+          <FormHelperText error>{error}</FormHelperText> : null
+      }
+    </FormControl>);
+  };
+
+  validateEnoughBOT = (address) => {
+    const { walletAddresses, eventEscrowAmount, intl } = this.props;
+
+    for (let i = 0; i < walletAddresses.length; i++) {
+      const checkingAddress = walletAddresses[i];
+      if (checkingAddress.address === address && checkingAddress.bot < eventEscrowAmount) {
```

Comment:
use `_.filter(walletAddresses, { address })` to get the walletAddress so you dont have to do a loop.